compiler/z/codegen/ControlFlowEvaluator.cpp

```diff
@@ -466,13 +473,13 @@ OMR::Z::TreeEvaluator::lmaxEvaluator(TR::Node* node, TR::CodeGenerator* cg)
 TR::Register*
 OMR::Z::TreeEvaluator::fmaxEvaluator(TR::Node* node, TR::CodeGenerator* cg)
    {
-   return xmaxxminHelper(node, cg, TR::InstOpCode::CEBR, TR::InstOpCode::COND_BHR, TR::InstOpCode::LER);
+   return xmaxxminHelper(node, cg, TR::InstOpCode::CEBR, TR::InstOpCode::COND_MASK10, TR::InstOpCode::LER);
```

Comment:
Is it immediately obvious to everyone that `COND_MASK10` means "equal or low" and that `COND_MASK12` means "equal or high"?  Personally, I had to look it up, so I think a comment wouldn't hurt.  Same comment with the other `{f|d}{max|min}Evaluator` methods.

Reply:
> Finally, as tests are still failing on z/OS, please remove the "Closes: https://github.com/eclipse/omr/issues/5157" comment.

Oops!  I see that the problem on z/OS is simply with the TRIL parser, not with the implementations of the evaluators.  You can leave the "Closes:  #5157" line in the commit comment.

Reply:
Thanks for the suggestions @hzongaro. I have adjusted the commit title and body for the commits, along with adding comments for each of the evaluators. Please let me know if they look okay, otherwise I can make more changes. I left the "Closes: https://github.com/eclipse/omr/issues/5157" in the commit as you suggested.